apps/arena/lib/arena/game_updater.ex

```diff
@@ -485,6 +488,38 @@ defmodule Arena.GameUpdater do
     |> Map.put(:aditional_info, entity |> Entities.maybe_add_custom_info())
   end
 
+  defp report_game_results(state, winner_id) do
+    results =
+      Enum.filter(state.game_state.client_to_player_map, fn {client_id, _player_id} ->
+        Enum.member?(state.clients, client_id)
+      end)
+      |> Enum.map(fn {client_id, player_id} ->
+        player = Map.get(state.game_state.players, player_id)
+
+        %{
+          user_id: client_id,
+          ## TODO: way to track `abandon`, currently a bot taking over will endup with a result
+          result: if(player.id == winner_id, do: "win", else: "loss"),
+          kills: player.aditional_info.kill_count,
+          ## TODO: this only works because you can only die once
+          deaths: if(Player.alive?(player), do: 0, else: 1),
+          character: player.aditional_info.character_name
+        }
+      end)
+
+    payload = Jason.encode!(%{match_id: Ecto.UUID.generate(), results: results})
+
+    ## TODO: we should be doing this in a better way, both the url and the actual request
+    ## maybe a separate GenServer that gets the results and tries to send them to the server?
+    ## This way if it fails we can retry or something
+    spawn(fn ->
+      gateway_url = System.get_env("GATEWAY_URL") || "http://localhost:4001"
```

Comment:
should this be in the configuration module?

Reply:
Yes, changed [622cd6d](https://github.com/lambdaclass/mirra_backend/pull/529/commits/622cd6d101e20b92d86fde749ce12600b3598d30)